server/chat.js

```diff
@@ -215,7 +215,7 @@ Chat.namefilter = function (name, user, forStatus = false) {
 		if (name.includes('@') && name.includes('.')) return '';
 
 		// url
-		if (/[a-z0-9]\.(com|net|org)/.test(name)) name = name.replace(/\./g, '');
+		if (/[a-z0-9]\.(com|us|uk|co|gg|tk|ml|gq|ga|xxx|download|stream|net|org|)/.test(name)) name = name.replace(/\./g, '');
```

Comment:
```suggestion
		if (/[a-z0-9]\.(com|us|uk|co|gg|tk|ml|gq|ga|xxx|download|stream|net|org|)\b/.test(name)) name = name.replace(/\./g, '');
```

`\b` is word boundary, right?